docs/installation.md

```diff
@@ -110,3 +110,36 @@ The build directory has a structure that mirrors the source directory, and you
 can type `make` in any one of its subdirectories to do partial builds. In
 practice, though, it's safest to always build from the top of the build tree.
 
+
+## Preinstalled PETSc for RDycore on certain DOE machines
+
```

Comment:
This is great! Thanks for adding this section.

Reply:
Do you think it's worth adding a README.md file to the `config/` directory briefly describing its contents, including the `install.*` scripts? Maybe it's not necessary, but it's nice to write this stuff down while it's in one's head.

Reply:
Good idea of adding a README.md. I will do that.